assets/src/edit-story/components/modal/index.js

```diff
@@ -46,15 +46,20 @@ export const GlobalStyle = createGlobalStyle`
 
 	.WebStories_ReactModal__Overlay {
 		position: fixed;
-		top: ${ ADMIN_TOOLBAR_HEIGHT }px;
-		left: ${ ADMIN_MENU_WIDTH }px;
+		top: 0;
+		left: 0;
 		right: 0;
 		bottom: 0;
 		background-color: ${ ( { theme } ) => theme.colors.bg.v1 };
 		z-index: 10;
 	}
 
-	body.folded .WebStories_ReactModal__Overlay {
+	.edit-story .WebStories_ReactModal__Overlay {
```

Comment:
What does this mean? Is there a `ReactModal__Overlay` outside `edit-story`?

Reply:
When running the editor within WordPress, the modal needs to have some offset in order to not overlay the admin menu & toolbar at the top.

When running the editor standalone within the storybook, this offset is not needed. `.edit-story` is used as the selector to distinguish the two scenarios.

Reply:
Hmm. This has a couple of big drawbacks: 

1. We're at mercy of WP layout.
2. We're less universal and more coupled to WP.

Would it be better to supply a "modal" overlay on top of the editor and positioned `absolute`? We're doing this for `Moveable` and numerous other cases. We could even try to make the current `InOverlay` more universal and reuse some code.

Reply:
Looks like `react-modal` supports a `parentSelector=` prop that allows moving the overlay on top of the editor instead of outside `#edit-story`

Reply:
Is it just `parentSelector`? Can we just pass a parent container?

Reply:
Doesn't look like it [according to the docs](http://reactcommunity.org/react-modal/#using-a-custom-parent-node).

It also shouldn't conflict with [`Modal.setAppElement`](http://reactcommunity.org/react-modal/accessibility/#app-element) which is used  for a11y

Reply:
Just re-read this and think there was some confusion.

`.edit-story` is a class name that is applied to the `body` in WordPress admin. This is different from the `#edit-story` app root.

I'll change the selector to clarify this.